doc/api/n-api.md

```diff
@@ -1331,6 +1331,38 @@ JavaScript TypedArray Objects are described in
 [Section 22.2](https://tc39.github.io/ecma262/#sec-typedarray-objects)
 of the ECMAScript Language Specification.
 
+
+#### *napi_create_dataview*
+<!-- YAML
+added: v8.0.0
+-->
+```C
+napi_status napi_create_dataview(napi_env env,
+                                   size_t length,
+                                   napi_value arraybuffer,
+                                   size_t byte_offset,
+                                   napi_value* result)
+
+```
+
+- `[in] env`: The environment that the API is invoked under.
+- `[in] length`: Number of elements in the DataView.
+- `[in] arraybuffer`: ArrayBuffer underlying the DataView.
+- `[in] byte_offset`: The byte offset within the ArrayBuffer from which to
+start projecting the DataVIew.
```

Comment:
typo: `DataView` (also, I’d indent this line to be on the same indentation as `` `[in] …``)